packages/utilities/src/guards/Rate Limiter/RateLimit.ts

```diff
@@ -23,15 +24,15 @@ export function RateLimit<T extends CommandInteraction | SimpleCommandMessage>(
   timeout: TIME_UNIT,
   value: number,
   options: RateLimitOption<T> = {
-    ephemeral: false,
-    message: "message being rate limited!, please try again at {until}",
+    ephemeral: true,
```

Comment:
not sure why `ephemeral` was changed to `true`. but sure i guess

Reply:
as @ravener mentioned reasons in PR. I also believe, it is good this way.